app/javascript/shared/components/ui/dropdown/AddLabel.vue

```diff
@@ -1,34 +1,34 @@
 <template>
-  <woot-button
-    variant="hollow"
-    size="tiny"
-    icon="ion-plus-round"
-    color-scheme="secondary"
-    class-names="button-wrap"
-    @click="toggleLabels"
-  >
-    {{ $t('CONTACT_PANEL.LABELS.MODAL.ADD_BUTTON') }}
-  </woot-button>
+  <span role="button" class="label--add" @click="addLabel">
```

Comment:
Please refrain from @click listeners on non-button attributes. Otherwise, we will have to handle keyboard listeners, tabIndex etc here.

Reply:
@pranavrajs Done.